### DISCUSSION TITLE
Custom configuration parameters for Z-Wave JS

### DISCUSSION THREAD

**Discussion by raym777** [zwave-js/zwave-js#2781] [General] [answered]
I raised [this issue](https://github.com/home-assistant/core/issues/51458) on the homeassistant/core repo yesterday, relating to not being able to set custom (or missing) cofig params on nodes in Z-Wave JS from HA. 

Based on the feedback received so far, it appears `HA has never supported custom configuration parameters for Z-Wave JS` and that I `should work with the node-zwave-js project to have these parameters added to the device config files so they can be supported in HA`.

I'm happy to do this and contribute in a resolution any way I can. Please advise.

**Comment by AlCalzone** [comment, marked as answer]
@blhoward2 has reworked all the Aeotec configuration files based on data sheets he got from Aeotec. They should be correct, but there's always the possibility we overlooked something.

We'll need to look at this on a case by case basis. Please open a separate issue for each device (bug report) and share the following info for each device:
1. Device label
2. manufacturer ID, product type and product ID
3. firmware version
4. Which parameters are missing

2./3. can be determined from a driver log (loglevel debug), but that shouldn't be necessary if you can pull that from the UI (e.g. zwavejs2mqtt).

**Reply by raym777** [reply]
Thank you. I've raised the issue [here](https://github.com/zwave-js/node-zwave-js/issues/2796).
